lib/ecto/adapters/postgres/connection.ex

```diff
@@ -494,6 +493,7 @@ if Code.ensure_loaded?(Postgrex.Connection) do
       |> List.flatten
       |> Enum.filter(&(&1 != nil))
       |> Enum.join(" ")
+      |> String.rstrip
```

Comment:
Can we remove the need for this `rstrip`? It is a bit expensive as it needs to traverse the whole string, so I think we should find a way to not need it.